### PR TITLE
vscode: update to 1.60.1

### PR DESCRIPTION
**Pull request by atk**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/vscode/template [hunk 1]
```diff
@@ -11,60 +11,60 @@ maintainer="shizonic <realtiaz@gmail.com>"
 license="MIT"
 homepage="https://code.visualstudio.com/"
 distfiles="https://github.com/Microsoft/vscode/archive/${version}.tar.gz"
-checksum=3f0cde58ab558cd227c61900222f4363589a81e4df2fac2bb2042229c90a7ee8
+checksum=c3c7f10c19b76465a63c6179762187e53ddf1f3cee85f82b2ec447c9b6e9407a
 
 if [ "$XBPS_TARGET_WORDSIZE" = "32" ]; then
-	broken="FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory"
+    broken="FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory"
```
Comment (ericonr) [inline srcpkgs/vscode/template:17]: Please revert indentation change, it should be tab.

Comment (ericonr) [reply srcpkgs/vscode/template:17]: And really in all other places, I think your editor had a field day :p

**Comment by paper42** [comment]
Could you add the email you are using for commits to your GitHub profile? GitHub will then be able to properly link your profile from the commit.

File: srcpkgs/vscode/template (hunk 1 above)
Comment (atk) [reply srcpkgs/vscode/template]: Just did it. The new version turned the autoformatter on by default, it seems.

**Comment by atk** [comment]
> Could you add the email you are using for commits to your GitHub profile? GitHub will then be able to properly link your profile from the commit.

Did it for the next commit.

**Comment by paper42** [comment]
> > Could you add the email you are using for commits to your GitHub profile? GitHub will then be able to properly link your profile from the commit.
> 
> Did it for the next commit.

For some reason the commit author is still not clickable on GitHub for me.

**Pull request merged**